fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/BardLoggingFilter.java

```diff
@@ -199,6 +205,30 @@ public void filter(ClientRequestContext request, ClientResponseContext response)
     }
 
     /**
+     * Add a request id to pass to druid. Only added if x-request-id follows the format specified at:
+     * https://devcenter.heroku.com/articles/http-request-id
+     *
+     * @param requestId  Request id to add as queryId prefix to druid
+     */
+    private void appendRequestId(String requestId) {
+        if (isValidRequestId(requestId)) {
```

Comment:
Good catch. I think the _behavior_ is correct (yay testing) the name is just wrong. @DeathByTape, could you open a quick PR to correct this?